src/python/pants/task/task.py

```diff
@@ -144,9 +139,8 @@ def alternate_target_roots(cls, options, address_mapper, build_graph):
     """
 
   @classmethod
-  def _prepare(cls, options, round_manager):
+  def invoke_prepare(cls, options, round_manager):
     # Subclasses should not generally need to override this method.
```

Comment:
Er, sorry: I didn't mean remove the method entirely. Just the verbiage that implies that overriding it might be necessary in some cases.